### ISSUE TITLE
OSD volume text in upper right or lower left corner

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#357]
Originally reported on Google Code with ID 358

```
Describe your system:
 - Operating System (be specific): Windows XP SP3
 - Machine type (32-bit or 64-bit): 32-bit
 - Mupen64Plus version: 1.99.3
 - Plugins used: Mupen64Plus OpenGL Video Plugin by Rice v1.99.3,
Mupen64Plus SDL Audio Plugin v1.99.3, Mupen64Plus SDL Input Plugin
v1.99.3, Hacktarux/Azimer High-Level Emulation RSP Plugin v1.99.3


Not a major bug at all but anyway.


If you fast forward at the same time as you change the volume (including
mute) the volume text will take the place of the "Fast Forward" (upper
right corner) text instead if being in the middle of the screen as it used
to. It's also possible to get the volume text in the lower left corner.


How to reproduce it for upper right corner:
Start the emulator, press any of the volume buttons
(increase/decrease/mute), let the text fade away, hold down "Fast Forward"
and press any of the volume buttons again.

To get in in the lower left do the same but before you hold down Fast
Forward press some of the savestate buttons (like 0-9 to select a slot),
and then hold down Fast Forward (before the savestate text has faded away!)
and change the volume.


I attached an image showing the lower right corner effect.
```

Reported by `q.guran` on 2010-03-24 17:16:03

<hr>
- _Attachment: [volume_fast_forward.jpg](https://storage.googleapis.com/google-code-attachments/mupen64plus/issue-358/comment-0/volume_fast_forward.jpg)_

**Comment by richard42** [comment]
Reported by `richard42g` on 2011-05-22 16:10:33
- **Status changed**: `Accepted`
- **Labels added**: Type-Defect, Component-Core, Milestone-2.0

**Comment by richard42** [comment]
```
Just fixed a data corruption in commit 260:c1e0be49ef63 (currently only in my repo at
https://bitbucket.org/lazhur/ ). Could you please test if this also fixes your problem?
```

Reported by `sven@narfation.org` on 2011-07-09 20:10:35

**Comment by richard42** [comment]
```
It is merged in richard42's repositories
```

Reported by `sven@narfation.org` on 2011-08-29 14:44:15
- **Status changed**: `Fixed`
